### PR TITLE
build: install journal-remote units and directory regardless of GNUTLS

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -3794,6 +3794,7 @@ endif
 if HAVE_GNUTLS
 systemd_journal_remote_LDADD += \
 	$(GNUTLS_LIBS)
+endif
 
 # systemd-journal-remote make sense mostly with full crypto stack
```
Comment (vcaputo) [inline Makefile.am:3799]: your change somewhat contradicts this comment...

**Comment by mischief** [comment]
fixes coreos/bugs#928

File: Makefile.am (hunk 1 above)
Comment (mischief) [inline Makefile.am:3799]: are you suggesting the comment be deleted? :-)

Comment (vcaputo) [inline Makefile.am:3799]: If the change is appropriate then the comment should be updated to reflect the new reality, not left out of sync.  I'm unsure on the appropriateness of the change though, we should just get gnutls in.

Comment (mischief) [inline Makefile.am:3799]: unfortunately there seem to be 'concerns' about shipping gnutls (https://github.com/coreos/coreos-overlay/pull/1587#issuecomment-148422613), and i believe that the unit files and directory for remote journal should be created regardless of tls support.

**Comment by marineam** [comment]
LGTM since it was merged as-is upstream.

**Pull request merged**